content/language/nodejs/develop.md

```diff
@@ -196,6 +196,8 @@ Dockerfile.
 
 Also, add a new volume to the server service for the bind mount. For this application, you'll mount `./src` from your local machine to `/usr/src/app/src` in the container.
 
+Lastly, expose port `9229` for debugging.
```

Comment:
Let's use publish instead of expose, to match the CLI flag (as opposed to the dockerfile instruction name)

```suggestion
Lastly, publish port `9229` for debugging.
```